docs/reference/settings/audit-settings.asciidoc

```diff
@@ -19,9 +19,12 @@ Set to `true` to enable auditing on the node. The default value is `false`.
 Specifies where audit logs are output. For example: `[ index, logfile ]`. The
 default value is `logfile`, which puts the auditing events in a dedicated
 file named `<clustername>_audit.log` on each node.
++
 You can also specify `index`, which puts the auditing events in an {es} index
 that is prefixed with `.security_audit_log`. The index can reside on the same
-cluster or a separate cluster.
+cluster or a separate cluster. deprecated[6.7.0, `logfile` will be the only audit
+option available in 7.0. A separate log file ingestion component should instead
+be used for indexing the audit trail.]
```

Comment:
This deprecation comment confused me on my earlier review, because the text talks about "logfile" but not "output".
It probably reads OK in context, but I would have phrased it more like:

> The `outputs` setting will be removed in 7.0 as there will only be one supported output type (`logfile`). Users who wish to store their audit information in an Elasticsearch index should write to the file output, and a use a file ingestion component to index it into Elasticsearch.

Entirely up to you whether you want to update it.

Reply:
You're right, I have made the change as suggested! thanks 👍